ui/ClipboardImporter.js

```diff
@@ -90,8 +90,37 @@ class ClipboardImporter extends HTMLImporter {
     // specific format, e.g., id="docs-internal-guid-5bea85da-43dc-fb06-e327-00c1c6576cf7"
     let bold = body.find('b')
     if (bold && /^docs-internal/.exec(bold.id)) {
-      return bold
+      body = bold
     }
+
+    body.findAll('span').forEach(span => {
+      const style = span.el.style
```

Comment:
In general you should use span.getNativeElement() instead of span.el as  does not work in the memory DOM.